ADAL/src/request/ADWebAuthResponse.m

```diff
@@ -106,6 +106,7 @@ - (void)handleResponse:(ADWebResponse *)webResponse
                 [_responseDictionary setObject:rawResponse
                                         forKey:@"raw_response"];
                 SAFE_ARC_RELEASE(rawResponse);
+                completionBlock(_responseDictionary);
                 break;
```

Comment:
Make this a return; not a break; then. Otherwise someone might change the subsequent code later and cause other regressions.